bundles/org.openhab.binding.touchwand/README.md

```diff
@@ -40,6 +41,12 @@ After adding TouchWand Hub the auto discovery will add all switches dimmers and
 
 No thing configuration is needed
 
+## AlarmSensor channels 
+
+**Touchwand Wanderfull™** supports various types of alarm sensors for example water leak sensors , door/window sensor , motion sensor.
+Trying to develop a generic Alarm Sensor support all AlarmSensor types has all the channels avaible but only the relevant ones will updates with the relevant values.
+It is the user responsibility to use the proper channels for the sensors.
+
```

Comment:
It is standard in other bindings to make a table of possible channels here that includes their item types and descriptions.

Reply:
done